src/ui/components/inputs/BaseTextInput.tsx

```diff
@@ -89,6 +89,7 @@ const StyledTextInput = styled(RNTextInput).attrs(({ theme }) => ({
     padding: 0,
     height: '100%',
     ...inputStyle,
+    color: editable ? theme.colors.black : theme.colors.greyDark,
```

Comment:
Pour ne pas override la valeur défini dans inputStyle quand l'input n'est pas disabled :
```suggestion
    color: editable ? inputStyle.color : theme.colors.greyDark,
```